core/tests/puppeteer-acceptance-tests/blog-admin-tests/assign-roles-to-users-and-change-tag-properties.js

```diff
@@ -17,6 +17,10 @@
  * tag properties from the blog-admin page.
  */
 
+// TODO(#16552): user-story(B8 cell) left as blocked by this issue.
```

Comment:
No one will know what this means -- please write it in terms such that any developer can understand what is needed, even those who haven't been in our team discussions..

Reply:
modified the comment